src/stan-users-guide/gaussian-processes.Rmd

```diff
@@ -1086,3 +1151,98 @@ generated quantities {
+
+$$
+K(x, x' \mid \sigma, l) = \sigma_0^2 + (x', x)
+$$
+
+We can define a dot product covariance function almost identicle to defining the squared exponential covariance function, instead as `gp_dot_prod_cov` with a constant parameter $\sigma_0$, for training data `matrix[D] x[N]`, an $N\times D$ matrix as
+
+```
+matrix[N, N] K = gp_dot_prod_cov(x, sigma0)
+```
+
+and for the cross covariance matrix with $N_{pred} \times D$ `matrix[D] x[N_pred] x_pred` as:
+
+```
+matrix[N, N_pred] K gp_dot_prod_cov(x_pred, x, sigma0)
+
+```
+
+Fixing the bias parameter, we can generate samples from the linear kernel (left) and the linear kernel fitting a linear function:
+
+//////////////GRAPHIC
+
+The code for fitting a linear model in Stan with a Gaussian Process is almost indenticle to that of fitting a GP regression with a squared exponential kernel, but instead replacing every instance of `gp_exp_quad_cov` with `gp_dot_prod_cov` and declaring and using parameteres for the dot product kernel, accordingly.
+
+### The Periodic Covariance Function
+
+The periodic covariance function is used to interpolate periodic trends in data, and can be used to model anything that follows a periodic pattern, such as seasonal trends or oscillations in time. The parameterization of the periodic covariance function is similar to that of the squared expontential, but with the addition of one more parameter, a period parameter. It's common to fix the period based on application, and is difficult to estimate. For example, the period could be fixed to days in a year. The periodic covariance function, with times $t, t'$ as
+
+$$
+K(t, t' \mid \sigma, l, \rho) = \sigma^2 \exp\left(\dfrac{-2 sin^2(\pi(t - t')/ \rho)}{2 l^2}\right).
+$$
+
+For $\sigma > 0$, $l > 0$ , and $\rho > 0$. We use $t$ since it is common to see the periodic covariance function used in time series application. Typically, $t$ will denote points measured in time. If data is measured at evenly spaced time points, we might set $t$ as $\mathbf{t}=[1,2,3,...,T]'$ where $T$ is the maximum number of measured time points, $\mathbf{t}$ is a column vector and the apostrophe indictaces the transpose. If $t$ isn't measured at evenly spaced time points, $t$ need not necessarily be such that $t_i-t_{i+1}$ for  all $i=1,2,3,...,T$. It is common to rescale $\mathbf{t}$ so that the integral to approximate is compatible with Stan's sampler. In order to approximate the periodic function well, we might instead set $t$ as $s(t) = t \space mod \space \rho$, where $mod$ is the modulus operator. In that case the periodic covariance function becomes
```

Comment:
"It is common to rescale $\mathbf{t}$ so that the integral to approximate is compatible with Stan's sampler."
what does this mean?